Source/Core/AudioCommon/CubebStream.h

```diff
@@ -25,6 +28,7 @@ class CubebStream final : public SoundStream
   bool Init() override;
   bool SetRunning(bool running) override;
   void SetVolume(int) override;
+  static bool IsValid() { return true; }
```

Comment:
Shouldn't this have a symmetric one in the `#else` portion that returns `false` instead? Unless I missed something, this shouldn't compile correctly without `HAVE_CUBEB`, since there's no `CubebStream::IsValid` member in that case.

Reply:
If it isn't defined, then it will fall back to the `IsValid` definition in `SoundStream`:

https://github.com/dolphin-emu/dolphin/blob/56b7b0a804bfd63d3567ef51264e726d73f3a309/Source/Core/AudioCommon/SoundStream.h#L19

Reply:
Oh, great, I didn't realize it was a base class method. Too bad you can't slap `override` on a `static` method to make it explicit. Guess you can ignore both my comments then 👍